docs/developers/contributing/dev_install.md

```diff
@@ -82,7 +82,13 @@ In order to make changes to `napari`, you will need to [fork](https://docs.githu
 
     Note that in this last case you will need to install your Qt backend separately.
 
-5. We use [`pre-commit`](https://pre-commit.com) to format code with
+    ```{note}
+    In all of the above cases, you may wish to include the `optional` dependency group,
+    in addition to `dev`. It includes `numba` and other performance packages that are
+    included with the typical end-user `napari[all]` installation.
+    ```
```

Comment:
Because I am not myself very familiar with the "dependency group" jargon, I would like to suggest using an example to make it clearer:

````suggestion
    ```{note}
    In all of the above cases, you may wish to include the `optional` dependency group,
    in addition to `dev`, by using `pip install -e ".[dev, optional]"`, for example.
    The `optional` dependency group includes `numba` and other performance packages that are
    included with the typical end-user `napari[all]` installation.
    ```
````